Enterprise-CMCS/macpro-mdct-qmr
# MDCT-2335: Measurement Date Range Enhancements

### Description



### Related ticket(s)


MDCT-2335

---
### How to test

- [Login](https://d11kxfufarjuyj.cloudfront.net/)
- Start with 2021, open a measure in adult, child, and health home core sets
- Confirm that the "Date Range" section now has a radio button option
- Confirm that when you click no, you see a nested date range (check before and after to see UI difference)
- Repeat for 2022 and 2023

| Before  | After |
| ------------- | ------------- |
| ![image-20230327-145438](https://github.com/Enterprise-CMCS/macpro-mdct-qmr/assets/6752256/12bffe1b-14b9-47fb-97c4-f9368707c273) | ![Screenshot 2023-06-13 at 7 46 04 PM](https://github.com/Enterprise-CMCS/macpro-mdct-qmr/assets/6752256/119f59cf-4655-457f-adf2-5612e5e14463)  |

Validations:
**When a user doesn't pick either radio button option:**
<img width="1062" alt="Screenshot 2023-06-21 at 1 18 38 PM" src="https://github.com/Enterprise-CMCS/macpro-mdct-qmr/assets/6752256/9e7cff7c-a59b-471e-8e0d-433d38879ff5">

<img width="1257" alt="Screenshot 2023-06-21 at 1 18 20 PM" src="https://github.com/Enterprise-CMCS/macpro-mdct-qmr/assets/6752256/b5e29e8b-8ea4-4838-8dc4-1b03793a9b5c">

**When a user picks "no" and leaves the nested date range blank:**
![image](https://github.com/Enterprise-CMCS/macpro-mdct-qmr/assets/6752256/15d07d18-4363-4999-9419-27eba9dfc618)
![Screenshot 2023-06-20 at 10 41 49 AM](https://github.com/Enterprise-CMCS/macpro-mdct-qmr/assets/6752256/aad72bf4-6d95-4535-a6da-a5ba365ec9ed)

### Important updates




---
### Author checklist


---


_convert to a different template: [test → val](?expand=1&template=test-to-val-deployment.md)_ | _[val → prod](?expand=1&template=val-to-prod-deployment.md)_
